### ISSUE TITLE
The structure of the different headings is incorrect

### ISSUE THREAD

**Issue by sabderemane** [pypi/warehouse#18162]
Hi, I have seen an issue in the ordering of the headings in the project which impact all project under the warehouse. This is an issue in terms of accessibility. 

"Navigation" should be a heading level 2 according the structure of the website. 

"Project link" is a heading level 6 but I can't see any heading level 4 and 5 before, it should be a heading just under "Verified details", which is a heading level 3 for now. The same applies for the similar headings as showed in the screenshot with the tool I have used.

---

**Screenshot**: 

![Image](https://github.com/user-attachments/assets/9564b35e-3f87-45ac-b5b4-b6b511662f37)

**Comment by miketheman** [comment]
Thanks for reporting @sabderemane ! Can you share some details of the tool you're using that shows the heading, so any fixes could be evaluated the same way?

**Comment by sabderemane** [comment]
Sure @miketheman ! I'm using [HeadingsMap on Google Chrome](https://chromewebstore.google.com/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi) here but [HeadingsMap also exist for Firefox](https://microsoftedge.microsoft.com/addons/detail/headingsmap/bokekiiaddinealohkmhjcgfanndmcgo) and [Microsoft Edge](https://microsoftedge.microsoft.com/addons/detail/headingsmap/bokekiiaddinealohkmhjcgfanndmcgo). 
It will show the headings structure of the website has it should be rendered for assistive tools. It shows in red any headings that are inconsistent with previous headings (or potentially missing headings).

**Comment by mashhoorahdal** [comment]
Fixed the issue, [PR](https://github.com/pypi/warehouse/pull/19460)

**Comment by dhruvatr** [comment]
Hi, I Would like to work on this Issue. Can you assign it to me?

**Comment by 47combinator** [comment]
Hi! I’m a new contributor and I’d like to work on fixing the heading hierarchy for accessibility if the issue is still open. Please let me know if I can take it.

**Comment by parashardiwas** [comment]
Hi! I'm a beginner to open source but I would love to tackle this issue. Could I please be assigned to it, and could you point me to the specific HTML template files where these headings are located?

**Comment by nlhkabu** [comment]
Closed by https://github.com/pypi/warehouse/pull/20223
Opening ticket under #19950 to retest accessibility including heading structure
